blueocean-dashboard/src/main/js/components/PipelinePage.jsx

```diff
@@ -44,17 +52,18 @@ export default class PipelinePage extends Component {
                         <TabLink to="/pr">Pull Requests</TabLink>
                     </PageTabs>
                 </PageHeader>
-                {React.cloneElement(this.props.children, { pipeline })}
+                {this.props.children}
             </Page>
         );
     }
 }
 
 PipelinePage.propTypes = {
-    children: object,
+    children: PropTypes.any,
```

Comment:
I believe @sophistifunk had made a point as well that using the PropTypes directly via destructuring will cause problems with Flow, which we may end up integrating into our plugins once we sort out any remaining kinks in the JDL.